README.md

```diff
@@ -20,26 +20,31 @@ Or install it yourself as:
 
 ## Usage
 
-### Sinatra instructions
-#### App setup
-* Add to Gemfile as well as listen 3.1.1
-* Require in server file (i.e. config.ru)
```

Comment:
ok, I thought I had to add it to some server side ruby so that opal server would serve it up in the asset pipeline, but if you're saying you didn't need to do it, then it's fine